nj-core/src/worker.rs

```diff
@@ -133,7 +133,6 @@ pub trait JSWorker: Sized + Send + 'static {
     }
 
     /// call by Node to create promise
-    #[no_mangle]
```

Comment:
I think https://github.com/rust-lang/rust/issues/28179 is what's going on. `cargo check` says that this will throw a hard error in the future but it's just a warning for now so I removed it.